cns/middlewares/k8sSwiftV2_windows.go

```diff
@@ -58,3 +62,55 @@ func (k *K8sSWIFTv2Middleware) addDefaultRoute(podIPInfo *cns.PodIpInfo, gwIP st
 	}
 	podIPInfo.Routes = append(podIPInfo.Routes, route)
 }
+
+// append the default deny acl's to the list defaultDenyACL field in podIpInfo
+func addDefaultDenyACL(podIPInfo *cns.PodIpInfo) error {
+	blockEgressACL, err := getDefaultDenyACLPolicy(hcn.DirectionTypeOut)
+	if err != nil {
+		return errors.Wrap(err, "Failed to create default deny ACL policy egress")
```

Comment:
error strings are not sentences, they're just fragments that can be embedded and strung together - don't capitalize 
```suggestion
		return errors.Wrap(err, "failed to create default deny ACL policy egress")
```

Reply:
lowercased it thanks